packages-web/badge-button/src/BadgeButton.tsx

```diff
@@ -0,0 +1,25 @@
+import { createElement, useCallback, ReactNode } from "react";
+import { hot } from "react-hot-loader/root";
+
+import { BadgeButton as BadgeButtonComponent } from "./components/BadgeButton";
+import { BadgeButtonContainerProps } from "../typings/BadgeButtonProps";
+import { executeAction } from "@widgets-resources/piw-utils";
+
+const BadgeButton = (props: BadgeButtonContainerProps): ReactNode => {
+    const onClick = useCallback(() => {
+        executeAction(props.onClickEvent);
+    }, [props.onClickEvent]);
+
+    return (
+        <BadgeButtonComponent
+            bootstrapStyle={props.bootstrapStyle}
+            className={props.class}
+            label={props.label && props.label.status === "available" ? props.label.value : ""}
+            onClickAction={onClick}
```

Comment:
should we align this name `onClick={onClick}` ?

Reply:
Done